adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/WEB-INF/views/responsive/pages/checkout/multi/3ds_payment.jsp

```diff
@@ -0,0 +1,48 @@
+<%@ page trimDirectiveWhitespaces="true" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<html>
+<head>
+    <script src="https://${checkoutShopperHost}/checkoutshopper/sdk/4.3.1/adyen.js"
+            integrity="sha384-eNk32fgfYxvzNLyV19j4SLSHPQdLNR+iUS1t/D7rO4gwvbHrj6y77oJLZI7ikzBH"
```

Comment:
do we need integrity?

Reply:
it is recommended in our (docs)[https://docs.adyen.com/online-payments/components-web?tab=script_2#step-2-add-components] so i added it